src/cascadia/TerminalControl/UiaTextRange.cpp

```diff
@@ -143,7 +82,7 @@ IFACEMETHODIMP UiaTextRange::Clone(_Outptr_result_maybenull_ ITextRangeProvider*
     *ppRetVal = nullptr;
     try
     {
-        *ppRetVal = new UiaTextRange(*this);
+        *ppRetVal = WRL::Make<UiaTextRange>(*this).Detach();

```

Comment:
If you switch to `MakeAndInitialize`, you can pass `ppRetVal` directly in as `WRL::MakeAndInitialize<UiaTextRange>(ppRetVal, *this)`

Reply:
and then you can RETURN_IF_FAILED it.